docs/application/dotnet/guides/nui/nui-components/Pagination.md

```diff
@@ -1,67 +1,51 @@
-# Pagination
+---
+keyword: pagination, indicator, property, style, Pagination, image, example, page, Class, key, KeyEvent
+---
 
+# Pagination
 
-Pagination is a component for making navigation through multipage windows more user-friendly.
-It can be used to display the number of available pages and marking the currently viewed one.
-The figure below illustrates the use of a simple pagination - it shows 5 available pages, of which the first one is currently displayed.
+Pagination is a component for making navigation through multipage windows more user-friendly. It can be used to display the number of available pages and marking the currently viewed one. The following figure illustrates the use of simple pagination. It shows 5 available pages, of which the first one is currently displayed.
```

Comment:
Styleguide: when we have following in sentence, the ending sentence must end with : not . 
```suggestion
Pagination is a component for making navigation through multipage windows more user-friendly. It can be used to display the number of available pages and marking the viewed ones. The following figure illustrates the use of simple pagination with 5 available pages, of which the first one is displayed currently:

Reply:
it should be "the currently viewed one"
it doesn't mark "the viewed ones"